### PR TITLE
Ios script fixes

### PR DESCRIPTION
**Pull request by marconett**
see https://forum.defold.com/t/building-custom-engine/66910

### CODE REVIEW

**Comment by dapetcu21** [comment]
Oh! I see you bumped into the same issue as me in #5382

File: scripts/mobile/ios-repack.sh [hunk 1]
```diff
@@ -109,7 +109,7 @@ TARGET="$(cd "$(dirname "${SOURCE}")"; pwd)/${APPLICATION}.repack"
     "${SECURITY}" cms -D -i "${PROFILE}" > "${PROVISION}"
     "${PLISTBUDDY}" -x -c 'Print :Entitlements' "${PROVISION}" > "${ENTITLEMENT}"
 
-    EXENAME=$(defaults read ${BUILD}/Payload/${APPLICATION}.app/Info.plist CFBundleExecutable)
+    EXENAME=$(defaults read ${BUILD}/Payload/"${APPLICATION}".app/Info.plist CFBundleExecutable)
```
Comment (JCash) [inline scripts/mobile/ios-repack.sh:112]: I would expect the entire path being wrapped by " ?
E.g. if the BUILD also contains spaces

File: scripts/package/package_xcode_and_sdks.sh [hunk 2]
```diff
@@ -90,12 +90,7 @@ function package_xcode() {
 
 	pushd ${XCODE}
 
-	EXTRA_ARGS="--exclude=${_name}/usr/bin --exclude=${_name}/Developer/Platforms --exclude=${_name}/usr/lib/sourcekitd.framework --exclude=${_name}/usr/metal"
-	for f in ${_name}/usr/lib/*.dylib
-	do
-		EXTRA_ARGS="--exclude=${f} ${EXTRA_ARGS}"
-	done
-
+	EXTRA_ARGS="--exclude=${_name}/Developer/Platforms --exclude=${_name}/usr/lib/sourcekitd.framework --exclude=${_name}/usr/metal"
```
Comment (JCash) [inline scripts/package/package_xcode_and_sdks.sh:93]: Thank you!

I'll add the second step later, where we'll also output the slimmed down version for linux builds.

**Review by JCash — APPROVED** [review]

**Comment by JCash** [comment]
A fix is coming with https://github.com/defold/defold/pull/5426

**Pull request closed without merging**